app/utils/link-component.js

```diff
@@ -1,13 +1,12 @@
 import LinkComponent from '@ember/routing/link-component';
 import {computed} from '@ember/object';
-import {invokeAction} from 'ember-invoke-action';
 
 LinkComponent.reopen({
     active: computed('attrs.params', '_routing.currentState', function () {
         let isActive = this._super(...arguments);
 
-        if (typeof this.get('alternateActive') === 'function') {
-            invokeAction(this, 'alternateActive', isActive);
+        if (typeof this.alternateActive === 'function') {
```

Comment:
Should this be using `this.get()` for consistency?

Reply:
yeah probably, I'll fix it 😄